modules/before-updating-web-console.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * updating/updating_a_cluster/updating-cluster-web-console.adoc
+
+:_content-type: CONCEPT
+[id="before-updating-web-console{context}"]
+= Before updating using the web console
+
+Before updating using the web console, consider the following:
+
+* Your etcd backup is recent enough in case your update fails and you must restore your cluster to a previous state.
+
+* You might experience an interruption during the update process if you run an Operator or you have configured any application with the pod disruption budget. If `minAvailable` is set to 1 in `PodDisruptionBudget`, the nodes are drained to apply pending machine configs which might block the eviction process. If several nodes are rebooted, all the pods might run on only one node, and the `PodDisruptionBudget` field can prevent the node drain.
+
+* You might need to update the cloud provider resources for the new release if your cluster uses manually maintained credentials.
+
+* You should provide the administrator acknowledgment and review the list of APIs that were removed with the latest Kubernetes release, and migrate any affected components to use the new API version.
```

Comment:
The outgoing:

> * Review the list of APIs that were removed in Kubernetes 1.27, migrate any affected components to use the new API version, and provide the administrator acknowledgment. For more information, see xref:../../updating/preparing_for_updates/updating-cluster-prepare.adoc#updating-cluster-prepare[Preparing to update to {product-title} 4.14].

was possibly more version-specific than we'd like, but I prefer it's opening that focuses on reviewing ack requests instead of providing ack approvals.  We want to reduce the chances that folks blindly click through the acks without actually performing the assessments that the acks are requesting.  Maybe:

* You should review administrator acknowledgement requests, take any recommended actions, and provide the acknowledgement when you are ready.

or something, to keep the opening focus on figuring out what you're being asked to ack?

Admin acks are also a subset of `Upgradeable=False` conditions, so I'm personally fine not discussing admin acks at all here, and instead asking the admin to address any `Upgradeable=False` conditions that the UI might be presenting.

Reply:
Updated! Thanks for the suggestion.